File: backup/day1/guessme.rb
#!/usr/bin/env python

from random import randint,seed

seed()
wflag = 0
predict = randint(1,1000)
chance = 1
MAX = 10

while chance <= MAX:
    value = input('Chance: %s\nEnter the value : '% chance)
    if value == predict:
	print "You won :) !!!!!!!!!!!"
	wflag = 1
        break
    elif value < predict:
	print "%s is lesser" % value
    elif value > predict:
	print "%s is greater" % value
    chance += 1

if wflag == 0:
    print "You lost, looser :(........."
    exit(1)

File: .history/older-than/day1/guessme.py
#!/usr/bin/env python
import random
rand = random.randint(1, 1000)

chance = 10
while chance >= 1:
    value = input('chance : %d\nEnter the number : ' % chance)

    if value == rand:
	print "You won :) !!!!!!!"
        exit(0)
    elif value < rand:
	print "%s : value is lesser" % value
    elif value > rand:
	print "%s : value is greater" % value
    chance -= 1

print "You lost, looser.... :(......."


